src/breaking-change.ts

```diff
@@ -116,19 +117,21 @@ async function processViaAutoRest(swaggerPath: string) {
 
 //main function
 export async function runScript() {
+  const pr = await devOps.createPullRequestProperties(cli.defaultConfig())
+
   // See whether script is in Travis CI context
   console.log(`isRunningInTravisCI: ${isRunningInTravisCI}`);
```

Comment:
what would we return if we were running in Travis?

Reply:
It will not be detected as PR. It's not plan to run in Travis-CI anymore.